SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -258,38 +258,69 @@ const FString ZoningAttribute = DefaultServerWorkerType.ToString();
 // A list of components clients require on top of any generated data components in order to handle non-authoritative actors correctly.
 const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_NON_AUTH_CLIENT_INTEREST = TArray<Worker_ComponentId>
 {
+	// Actor components
 	UNREAL_METADATA_COMPONENT_ID,
 	SPAWN_DATA_COMPONENT_ID,
 	RPCS_ON_ENTITY_CREATION_ID,
+
+	// Multicast RPCs
 	MULTICAST_RPCS_COMPONENT_ID,
-	NETMULTICAST_RPCS_COMPONENT_ID_LEGACY
+	NETMULTICAST_RPCS_COMPONENT_ID_LEGACY,
+
+	// Global state components
+	SINGLETON_MANAGER_COMPONENT_ID,
+	DEPLOYMENT_MAP_COMPONENT_ID,
+	STARTUP_ACTOR_MANAGER_COMPONENT_ID,
+	GSM_SHUTDOWN_COMPONENT_ID,
```

Comment:
This is only required by servers - but honestly it's not updated much/ever, so we can leave it also.